spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,15 +32,26 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            mate
+            profile::core::anaconda
             profile::core::common
             profile::core::debugutils
             profile::core::docker
             profile::core::docker::prune
             profile::core::ni_packages
+            profile::core::x2go
+            profile::ts::hexrot
+            profile::ts::nexusctio
```

Comment:
So testing directly for the inclusion of other profile classes is actually a bit of an anti-pattern.  The reason this is in here is there were no tests at all and I banged something out in a couple of minutes.  Testing for external module is fine.  Testing directly for profile classes makes it hard to refactor the profiles.  This is acceptable for the moment but its technical debt.

Reply:
Please remove the test for profile classes.  Testing for external classes is OK.

Reply:
removed the self class test